lib/pynput/keyboard/_xorg.py

```diff
@@ -165,15 +182,59 @@ class Key(enum.Enum):
     tab = KeyCode._from_symbol('Tab')
     up = KeyCode._from_symbol('Up')
 
+    iso_lock = KeyCode._from_ISO('Lock')
+    iso_level2_latch = KeyCode._from_ISO('Level2_Latch')
+    iso_level3_shift = KeyCode._from_ISO('Level3_Shift')
+    iso_level3_latch = KeyCode._from_ISO('Level3_Latch')
+    iso_level3_lock = KeyCode._from_ISO('Level3_Lock')
+    iso_group_shift = KeyCode._from_ISO('Group_Shift')
+    iso_group_latch = KeyCode._from_ISO('Group_Latch')
+    iso_group_lock = KeyCode._from_ISO('Group_Lock')
+    iso_next_group = KeyCode._from_ISO('Next_Group')
+    iso_next_group_lock = KeyCode._from_ISO('Next_Group_Lock')
+    iso_prev_group = KeyCode._from_ISO('Prev_Group')
+    iso_prev_group_lock = KeyCode._from_ISO('Prev_Group_Lock')
+    iso_first_group = KeyCode._from_ISO('First_Group')
+    iso_first_group_lock = KeyCode._from_ISO('First_Group_Lock')
+    iso_last_group = KeyCode._from_ISO('Last_Group')
+    iso_last_group_lock = KeyCode._from_ISO('Last_Group_Lock')
+    iso_left_tab = KeyCode._from_ISO('Left_Tab')
+    iso_move_line_up = KeyCode._from_ISO('Move_Line_Up')
+    iso_move_line_down = KeyCode._from_ISO('Move_Line_Down')
+    iso_partial_line_up = KeyCode._from_ISO('Partial_Line_Up')
+    iso_partial_line_down = KeyCode._from_ISO('Partial_Line_Down')
+    iso_partial_space_left = KeyCode._from_ISO('Partial_Space_Left')
+    iso_partial_space_right = KeyCode._from_ISO('Partial_Space_Right')
+    iso_set_margin_left = KeyCode._from_ISO('Set_Margin_Left')
+    iso_set_margin_right = KeyCode._from_ISO('Set_Margin_Right')
+    iso_release_margin_left = KeyCode._from_ISO('Release_Margin_Left')
+    iso_release_margin_right = KeyCode._from_ISO('Release_Margin_Right')
+    iso_release_both_margins = KeyCode._from_ISO('Release_Both_Margins')
+    iso_fast_cursor_left = KeyCode._from_ISO('Fast_Cursor_Left')
+    iso_fast_cursor_right = KeyCode._from_ISO('Fast_Cursor_Right')
+    iso_fast_cursor_up = KeyCode._from_ISO('Fast_Cursor_Up')
+    iso_fast_cursor_down = KeyCode._from_ISO('Fast_Cursor_Down')
+    iso_continuous_underline = KeyCode._from_ISO('Continuous_Underline')
+    iso_discontinuous_underline = KeyCode._from_ISO('Discontinuous_Underline')
+    iso_emphasize = KeyCode._from_ISO('Emphasize')
+    iso_center_object = KeyCode._from_ISO('Center_Object')
+
+    brightness_up = KeyCode._from_symbol('XF86_MonBrightnessUp')
```

Comment:
I appreciate that you have added these media and brightness codes, but I will not be able to merge this until matching codes have been added to the remaining backends. I do not mind looking them up though.

Reply:
By "remaining backends", do you mean recognizing`brightness_up` keys on other operating systems as well, like Windows?